<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>474</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Free Video Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/images/fvdownloader.png</normaloff>:/images/fvdownloader.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="CustomDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>245</width>
     <height>390</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50000</width>
     <height>50000</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="accessibleName">
    <string>Search pane</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Dock widget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="VideoPlayerWidget" name="dockFrame">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="VideoControl" name="videoControl" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>238</y>
          <width>230</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="DescriptionPanel" name="descriptionWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>248</y>
          <width>230</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="VideoProgressBar" name="videoProgress">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>229</width>
          <height>18</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/images/help_about.png</normaloff>:/images/help_about.png</iconset>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset>
     <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionSearchVideo">
   <property name="icon">
    <iconset>
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>Search video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoControl</class>
   <extends>QWidget</extends>
   <header>videocontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomDockWidget</class>
   <extends>QDockWidget</extends>
   <header>customdockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoPlayerWidget</class>
   <extends>QFrame</extends>
   <header>videoplayerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DescriptionPanel</class>
   <extends>QWidget</extends>
   <header>descriptionpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoProgressBar</class>
   <extends>QProgressBar</extends>
   <header>videoprogressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
 </connections>
</ui>
